src/synapse-plugins/calculator-plugin.vala

```diff
@@ -108,6 +114,11 @@ namespace Synapse {
                     if (solution != null) {
                         double d = double.parse (solution);
                         Result result = new Result (d, query.query_string);
+                        result.description = "%s\n%s".printf (
+                            result.text,
```

Comment:
`result.text` -> `result.tooltip_text` so it shows the correct text but copies the result.

Reply:
I used a different solution